content/en/flux/guides/webhook-receivers.md

```diff
@@ -132,6 +132,8 @@ spec:
   secretRef:
     name: webhook-token
   resources:
+    - kind: GitRepository
+      name: flux-system
```

Comment:
Please remove this. Adding flux-system which is a diffrent repo to the webapp will make Flux do the wrong thing.